cores/esp8266/LwipIntfDev.h

```diff
@@ -272,22 +274,41 @@ boolean LwipIntfDev<RawDev>::begin(const uint8_t* macAddress, const uint16_t mtu
         }
     }
 
-    if (_intrPin < 0
-        && !schedule_recurrent_function_us(
+    if (_intrPin < 0 && !_scheduled)
+    {
+        _scheduled = schedule_recurrent_function_us(
             [&]()
             {
+                if (!_started)
+                {
+                    _scheduled = false;
+                    return false;
+                }
                 this->handlePackets();
                 return true;
             },
-            100))
-    {
-        netif_remove(&_netif);
-        return false;
+            100);
+        if (!_scheduled)
+        {
+            netif_remove(&_netif);
```

Comment:
This should also be added
```cpp
RawDev::end();
_started = false;
```
also master-missing `RawDev::end();` should be added above where `return false` are.

Reply:
so `end()`

Reply:
(also lines  [234](https://github.com/esp8266/Arduino/pull/9023/files#diff-ced83b441ac52d743d09c392b44aa2a6f9ec68e68ca1f8fe788fdd74ae125daaR234), [247](https://github.com/esp8266/Arduino/pull/9023/files#diff-ced83b441ac52d743d09c392b44aa2a6f9ec68e68ca1f8fe788fdd74ae125daaR247) and [251](https://github.com/esp8266/Arduino/pull/9023/files#diff-ced83b441ac52d743d09c392b44aa2a6f9ec68e68ca1f8fe788fdd74ae125daaR251))

Reply:
this way?